shell/imports/server/accounts/ldap.js

```diff
@@ -76,7 +76,7 @@ LDAP.prototype.ldapCheck = function (db, options) {
       client = ldapjs.createClient({
         url: fullUrl,
         tlsOptions: {
-          ca: [_this.options.ldapsCertificate],
+          ca: [db.getLdapCaCert() || false],
```

Comment:
Is `false` a valid input here?  https://nodejs.org/dist/latest-v4.x/docs/api/tls.html#tls_tls_connect_options_callback seems to indicate it expects either a string, buffer or array of either of those.

Depending on how the implementation details work out, it seems like this maybe has the potential to break non-self-signed certs, by specifying an invalid cacert here?

Maybe this is better written as:

```js
const cert = db.getLdapCaCert();
const tlsOptions = {};
if (cert) {
  tlsOptions.ca = cert;
}
client = ldapjs.createClient({
    ...
    tlsOptions,
});
```